src/lms-ds-loader/edfi_lms_ds_loader/scripts/mssql/create_section_tables.sql

```diff
@@ -0,0 +1,61 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TABLE lms.LMSSection (
```

Comment:
The MetaEd output has `[]` qualifiers on all of the SQL Server output. I tend to strip that out except where necessary (`[PRIMARY]`, `[Name]`), which makes it easier to copy the MSSQL script to a PGSQL script.